lib/modern_searchlogic/column_conditions.rb

```diff
@@ -85,7 +94,11 @@ def searchlogic_suffix_condition_match(method_name)
             block: lambda do |*args|
               validate_argument_count!(arity, args.length) if arity >= 0
               arel_conditions = column_names.map { |n| instance_exec(n, *args, &method_block) }.reduce(:or)
-              where(arel_conditions)
+              if arel_conditions.class == ActiveRecord::Relation
+                arel_conditions
+              else
+                where(arel_conditions)
+              end
```

Comment:
WDYT about renaming `arel_conditions` to `conditions` and open the check to be `conditions.is_a?(ActiveRecord::Relation)` since this should try to work with subclasses of ActiveRecord::Relation.

Reply:
Yeah I like that change, just updated.